Internet Inconsistencies with R-Programming

pkalynan opened the pull request:
### Article checklist
**NOTE: (Please ensure that you have only one open issue + linked pull request at a time. This will ensure that we complete the article in a timely manner from inception to publishing.)**
#### Preliminary Checks - Formatting and Structure 
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**
- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**
- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
- [x] In your PR - add a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) -  [Grammarly](https://grammarly.com) 
- [x] Please place your article through a 3rd party plagiarism checker? We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**

### Contribution guidelines
For first-time contributors and for more details, **see [Contributing Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**

This fixes #1655

pkalynan commented:
This fixes #1655

srishilesh reviewed (changes requested):
Hey @pkalynan,

Thanks for the PR.

This article looks great. I have requested a few changes to improve the readability of the article. Please let me know when it's done, we can move on to the tech review.

A few suggestions:
- Add a hero image of 600x400
- Always start the markdown headers from `###`, not `####`.
- Make sure you check for grammar errors using Grammarly. I found a few when I ran the grammar check. Resolve them
- You must quote sentences when you reference it from somewhere else, or if it's not your original idea.
- Add image sources, if they're not screenshots of yours.
- It is advised to keep the paragraphs smaller, which improves the readability.

Make sure to go through all the comments and resolve them before you request for the next review.

Thank you.

pkalynan commented:
Hi @srishilesh 

Thank you for your revision request and correction details. 

I corrected my article to make sentences shorter and concise for the reader. A hero image was added with 600x400 measurements. Headers were corrected as per your advised comments. Anchors were included to link headers for the Table of Contents. Images are screen captures or screen snippets from my device, so I added _Screen capture_ underneath each image. Markdown quotations were removed and are changed to regular text as they were not taken from any source - they are in my own words. Markdown code snippets were sectioned off to appear more presentable for the reader. There are only a few markdown commands in my article that I thought were appropriate that could benefit the reader. 


Thank you.

srishilesh commented on articles/internet-inconsistencies-with-r-programming/index.md:
```diff
@@ -0,0 +1,288 @@
+
+Did you notice the domain names listed above are companies registered with `ARIN` and `CIRA` without any connection to `RIPE`?  
+
+Internet inconsistencies exist.
```
It is abruptly concluded. Could you give a summary of facts, which makes you say that "Internet inconsistencies exist"?

srishilesh commented on articles/internet-inconsistencies-with-r-programming/index.md:
```diff
@@ -0,0 +1,288 @@
+
+Internet inconsistencies exist.
+
+### Takeaways
```
Though you have added the takeaways, it would be great if you give a proper conclusion of what was discussed previously, before specifying the takeaways.

srishilesh reviewed (changes requested):
Hey @pkalynan,

Thank you for making the requested changes. The article looks really polished now.

I have made changes in formatting and flow, please have a look at them.

Also, I have specified 2 other minor suggestions. Please resolve the 2 suggestions. (Don't disturb the remaining sections - everything else looks good).

Also, you have missed to upload a hero image. Please do that.

Let me know when they're done, we can move on to the final review.

Thank you.

pkalynan commented:
Hi @srishilesh 

Thanks for pointing those errors out. I corrected them.

srishilesh reviewed (approved):
Hey @pkalynan,

Thank you for making the changes.

From the next time, make sure you upload hero image of 600x400.

Moving this for the final review.

Thank you.

pkalynan commented:
Hi @srishilesh 

The hero image was not exactly 600x400. Maybe around 582x381. Thank you for your extremely helpful guidance. I will use them next time.

hectorkambow commented:
plagiairsm check done

hectorkambow commented:
@pkalynan  great tutorial overall - Im sure the community will find it very helpful. 

Please do go the changes and suggestions made by your peer reviewer to apply any applicable to future articles - and be sure to check out our [resources page](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) (if you havent already) for more helpful links and tips to improve our overall writings. 

🚀
